Python/hamt.c

```diff
@@ -251,6 +251,29 @@ Further Reading
 3. Clojure's PersistentHashMap implementation:
    https://github.com/clojure/clojure/blob/master/src/jvm/
         clojure/lang/PersistentHashMap.java
```

Comment:
I think writing URL is time to break 79-columns rule.

Reply:
sure